barman/cli.py

```diff
@@ -52,6 +54,116 @@
 _logger = logging.getLogger(__name__)
 
 
+# Support aliases for argparse in python2
+if sys.version_info.major < 3:
+    class AliasedSubParsersAction(_SubParsersAction):
```

Comment:
Nice hack and thank you for putting the python2 workaround in.

This looks like it was mostly derived from https://gist.github.com/sampsyo/471779 - if so it's worth referencing that gist in a comment and including a note that the code is public domain. Although the CC0 1.0 license doesn't require attribution I think it's still important to note where the code came from just in case there are some issues with automated code scans in the future.